backend/pps/sch.py

```diff
@@ -0,0 +1,47 @@
+import math
+
+from backend.calc import CharacterStats
+from backend.pps.pps import HealerPps
+
+
+class SchPps(HealerPps):
+    r2_potency = 200
+    b3_potency = 290
+    bio_potency = 70
+    ed_potency = 100
+        
+    def get_pps(self, character_stats, caster_tax):
+        return self.total_potency_spreadsheet_port(character_stats, caster_tax) / self.get_cycle(caster_tax)
+        
+    # hard coded for a ~180 second cycle, actual length calculated by get_cycle
+    # todo: extend this for variable length
+    def total_potency_spreadsheet_port(self, character_stats, caster_tax):
```

Comment:
add ed per min as arg to this function -- the existing spreadsheet asks people. cuz you could heal with it instead (kek) and either way it's relevant for both pps and mp calculations

Reply:
sgtm, but i think it might be worth revisiting ed_per_min when we get around to a variable time pps calculation, since that's not going to be constant wrt encounter length.